xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxBorrowingServiceRaw.java

```diff
@@ -0,0 +1,140 @@
+package org.knowm.xchange.ftx.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.knowm.xchange.ftx.FtxExchange;
+import org.knowm.xchange.ftx.dto.account.FtxBorrowingInfoDto;
+import org.knowm.xchange.ftx.dto.account.FtxBorrowingRatesDto;
+import org.knowm.xchange.ftx.dto.account.FtxBorrowingsDto;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class FtxBorrowingServiceRaw extends FtxBaseService {
+
+  public FtxBorrowingServiceRaw(FtxExchange exchange) {
+    super(exchange);
+  }
+
+  public List<FtxBorrowingsDto> histories(String subaccount) {
+    try {
+      return ftx.getBorrowHistory(
+          exchange.getExchangeSpecification().getApiKey(),
+          exchange.getNonceFactory().createValue(),
+          signatureCreator,
+          subaccount
+      ).getResult();
+    } catch (IOException e) {
+      throw new FtxLendingServiceRaw.FtxLendingServiceException("Can't get lending infos subAccount: " + subaccount, e);
+    }
+  }
+
+  public List<FtxBorrowingsDto> histories(String subaccount, List<String> coins) {
```

Comment:
my comment about very similar signatures for histories, infos & rates was not addressed

Reply:
this signature FtxBorrowingsDto are created before my patch

Reply:
I change it ? for FtxBorrowingHistoryDto ?

Reply:
`  public List<FtxBorrowingInfoDto> infos(String subaccount, List<String> coins) {
    Objects.requireNonNull(coins);
    return infos(subaccount).stream()
        .filter(lendingInfo -> coins.contains(lendingInfo.getCoin()))
        .collect(Collectors.toList());
  }

  public List<FtxBorrowingInfoDto> infos(String subaccount, String... coins) {
    Objects.requireNonNull(coins);
    return infos(subaccount, Arrays.asList(coins));
  }
`

if you look at these functions thy are very similar, do we need them both?

Reply:
I removed all method with array, and I keep the list